project_milestone_status/readme/DESCRIPTION.md

```
@@ -0,0 +1 @@
+This module allows you to have the percentage of the execution of a project.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)